__tests__/app.test.js

```diff
@@ -156,9 +158,34 @@ describe("GET /api/users", () => {
             avatar_url: expect.any(String),
           });
         });
-        expect(body.users.length).toBe(4);
         expect(Array.isArray(body.users)).toBe(true);
+        expect(body.users.length).toBe(4);
+      });
+  });
+});
 
+describe.only("GET api/articles,", () => {
```

Comment:
Make sure you remove .only in tests or you may find you have hidden failing tests